docs/plugins/dynamic_policy_enforcement_points.mdpp

```diff
@@ -112,7 +113,7 @@ becomes:
 
     pep_api_data_obj_put_pre(*INSTANCE_NAME, *COMM, *DATAOBJINP, *BUFFER, *PORTAL_OPR_OUT)
 
-After `*INSTANCE_NAME`, note the next two types of this signature are serialized by the framework, but the last two are not.  This means that the information within `*COMM` and `*DATAOBJINP` will be available in the rule logic whereas `*BUFFER` and `*PORTAL_OPR_OUT` will explain they are not supported.
+After `*INSTANCE_NAME`, note the next three types of this signature are serialized by the framework, but the last one is not.  This means that the information within `*COMM`, `*DATAOBJINP`, and `*BUFFER` will be available in the rule logic whereas `*PORTAL_OPR_OUT` will explain they are not supported.
```

Comment:
"explain they are not" -> "explain it is not"

Reply:
Done.